### ISSUE TITLE
* 'aws_security_group.consul' error: : invalid or unknown key: egress

### ISSUE THREAD

**Issue by mickengland** [hashicorp/consul#1206]
Trying to follow the terraform guide at https://terraform.io/intro/getting-started/modules.html I get this error when running terraform plan.
- 'aws_security_group.consul' error: : invalid or unknown key: egress

I know outbound rules are only possible in VPC security groups. Is this trying to create a none VPC security by chance?

This is the content of my .tf file:

provider "aws" {
    access_key = "${var.access_key}"
    secret_key = "${var.secret_key}"
    region = "${var.region}"
}

module "consul" {
    source = "github.com/hashicorp/consul/terraform/aws"

```
key_name = "consul"
key_path = "/Users/mick/consul.pem"
region = "us-east-1"
servers = "3"
```

}

**Comment by mickengland** [comment]
I was able to get past this by commenting out these lines in consul.tf and loading the module locally rather than from github.

 // This is for outbound internet access
     // egress {
     //    from_port = 0
     //    to_port = 0
     //    protocol = "-1"
     //    cidr_blocks = ["0.0.0.0/0"]
     //}

After doing this I was still unable to launch in US east due to this error:

* Error launching source instance: The specified instance type can only be used in a VPC. A subnet ID or network interface ID is required to carry out the request. (VPCResourceNotSpecified)

After some digging it turned out my account is really old and uses ec2 classic by default. After switching to us-west-2 I was able to run apply with no problems but only if egress was commented out.
